cmd/prepare-storage.go

```diff
@@ -165,7 +165,7 @@ func prepForInitXL(firstDisk bool, sErrs []error, diskCount int) InitActions {
 	// Total disks unformatted are in quorum verify if we have some offline disks.
 	if disksUnformatted >= quorum {
 		// Some disks offline and some disks unformatted, wait for all of them to come online.
-		if disksUnformatted+disksOffline == diskCount {
+		if disksUnformatted+disksFormatted+disksOffline == diskCount {
 			return WaitForAll
 		}
 		// Some disks possibly corrupted and too many unformatted disks.
```

Comment:
> Should we return Abort even if too many disks are unformatted without checking if disksCorrupted is non-zero?

@krisis, in this case, if we have a corrupted disk then
`disksUnformatted+disksFormatted+disksOffline` won't be equal to `diskCount`